2023-09-XX-qubes-os-4-2-0-rc3-available-for-testing.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+title: "Qubes OS 4.2.0-rc3 is available for testing"
+categories: releases
+download_url: /downloads/
+---
+
+We're pleased to announce that the third [release candidate (RC)](#what-is-a-release-candidate) for Qubes OS 4.2.0 is now available for [testing](/doc/testing/). The ISO and associated [verification files](/security/verifying-signatures/) are available on the [downloads](/downloads/) page.
+
+## Explanation for the early RC
+
+We [announced RC2](/news/2023/08/28/qubes-os-4-2-0-rc2-available-for-testing/) less than a week ago. Normally, RC2 would have been tested for [approximately five weeks](/doc/version-scheme/#release-schedule) before we announced RC3. However, RC2 contained several bugs (listed below) that prevented certain users from testing it. These bugs have been fixed in RC3. We've decided to release RC3 early, as an exception to our usual policy, in order to get these fixes out as quickly as possible so that more users can test 4.2 for longer before the eventual stable release.
+
+## Main changes from RC2 to RC3
+
+- Fixed: ["Wi-Fi firmware missing from default templates on 4.2.0-rc2 ISO" (#8452)](https://github.com/QubesOS/qubes-issues/issues/8452)
+- Fixed: ["Qubes R4.2.0-rc2 cannot be installed on legacy BIOS system" (#8462)](https://github.com/QubesOS/qubes-issues/issues/8462)
+- Fixed: ["R4.2 (rc1, rc2) unable to boot on Thinkpad T430 when UEFI is enabled" (#8464)](https://github.com/QubesOS/qubes-issues/issues/8464)
```

Comment:
I ended up fixing https://github.com/QubesOS/qubes-issues/issues/8345 too

Reply:
This list is just the "main changes from RC2 to RC3," so I think it's fine to mention it here. :)